lib/rails_ext/action_text_storage_tracking.rb

```diff
@@ -0,0 +1,19 @@
+module ActionTextContentStorageTracking
+  def bytes_used
+    attachables.sum { |attachable| attachable.try(:byte_size) || 0 }
```

Comment:
I think we can use `attachable_filesize` here which tries both `byte_size` and `filesize`:

```suggestion
    attachables.sum { |attachable| attachable.attachable_filesize || 0 }
```